src/components/utils/useReducedMotion/pages/useReducedMotion.jsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
you don't need to create separate folder for `useReducedMotion`. Following the pattern we use:
```
utils/
├─ useReducedMotion.jsx
├─ useReducedMotion.stories.mdx
├─ pages/
│  ├─ reduced-motion.jsx

```